### PR TITLE
Fix query and images for gene list analysis

### PR DESCRIPTION
**Pull request by bgyori**
This PR fixes one of the queries for analysis. Since the downstream/upstream images were overflowing their container, I tried decreasing their size but this made them flow into the results list. @cthoyt if you can figure out how to avoid this, please push to this branch.

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -64,7 +64,7 @@ tests =
 web =
     flask
     flask-wtf
-    bootstrap-flask
+    bootstrap-flask<=2.0.0
```
Comment (bgyori) [inline setup.cfg:67]: We might want to add a lower bound as well depending on when Bootstrap4 was added - this didn't work with my 1.7.0 version.

Comment (bgyori) [reply setup.cfg:67]: Or did you actually mean to put `>=` here?

Comment (bgyori) [reply setup.cfg]: I concluded that the only logical explanation is that it should be >=

Comment (cthoyt) [reply setup.cfg]: Thanks for the typo spot, you were correct

**Pull request merged**